Automattic/jetpack
# My Jetpack: Licensing: Update href of view my plans button to go to Calypso Green

#### Changes proposed in this Pull Request:

* Updates location of "View my plans" button after activating a license to point to Calypso Green

Before this PR, this button sent the user to the My Plan section in the Jetpack dashboard. But, in the case of attaching a license to a site that only has standalone plugins, and no Jetpack, this presents an issue.

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Checkout branch
* Go to cloud.jetpack.com/pricing and make a purchase of any Jetpack product, other than CRM or Search
* After purchase, select configure manually at the bottom of the select and continue through until you can copy the license key
* On Jetpack site, set `define( 'JETPACK_ENABLE_MY_JETPACK_LICENSE', true );`
* Go to My Jetpack
* Click on "Activate a license" link
* Enter license
* Click "View my plans" button
* Ensure that you land on cloud.jetpack.com and can see your purchases
* Try the above with/without Jetpack installed

A screenshot of the success sce

<img width="1136" alt="Screen Shot 2022-05-17 at 3 27 40 PM" src="https://user-images.githubusercontent.com/1126811/168904779-c764f227-4f6f-449d-8228-34b486dd3420.png">

A screenshot of the purchases page in Calypso Green.

<img width="1392" alt="Screen Shot 2022-05-17 at 3 27 57 PM" src="https://user-images.githubusercontent.com/1126811/168904821-e9d6c170-e32f-49cd-b60a-c1a488b723ad.png">
